services/src/handlers/projects.rs

```diff
@@ -200,7 +200,7 @@ mod tests {
         }
 
         let options = ProjectListOptions {
-            owner: ProjectOwner::Any,
+            only_owned: false,
```

Comment:
What does _only owned_ mean? Is the intention something like _created by me_?

Reply:
the idea was to (similar to google docs) either show only the user's own projects (user has owner permission) or all projects she can access (at least read permission). So created_by_me doesn't fit. Maybe `require_owner_permission` or invert it `include_shared_projects`?